docs/data/charts/zoom-and-pan/ZoomBarChart.tsx

```diff
@@ -6,17 +6,14 @@ export default function ZoomBarChart() {
     <BarChartPro
       width={600}
       height={300}
-      zoom
-      series={[
+      xAxis={[
         {
-          label: 'Series A',
-          data: data.map((v) => v.y1),
-        },
-        {
-          label: 'Series B',
-          data: data.map((v) => v.y2),
+          scaleType: 'band',
+          data: data.map((v, i) => i),
+          zoom: true,
```

Comment:
On `Bar` and `Line` charts, when I add `{zoom: true}` it doesn't pick the default values for the axis, and I have to specifically add `scaleType|data`. Do we want to change this behaviour so we can simply enable zooming with `{zoom: true}` (for a future mr though)

Reply:
Good idea. I created #13794 to track it